features/site/home/CommunityFeatures.tsx

```
@@ -34,34 +34,38 @@ const Features = ({ items, ...props }: any) => {
-export const HomeExampleProps = () => {
+export const CommunityFeatures = () => {
   return (
     <ContainerLarge position="relative">
       <XStack px="$6" pt="$8" space="$4" $sm={{ flexDirection: 'column', px: 0 }}>
         <YStack w="50%" $sm={{ w: '100%' }}>
           <Features
             space="$4"
             items={[
-              <FeatureItem label="Press & hover events">
-                onHoverIn, onHoverOut, onPressIn, and onPressOut.
+              <FeatureItem label="Community Engagement">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)